polyfills/~html5-elements/config.json

```diff
@@ -16,5 +16,8 @@
 	"install": {
 		"module": "html5shiv",
 		"paths": ["dist/html5shiv.min.js"]
-	}
+	},
+	"dependencies": [
+		"Document"
+	]
```

Comment:
When running the bundled detect code in NodeJS, it will fail because `Document` does not exist. This change resolves that issue :-)